pg_search/src/telemetry/posthog.rs

```diff
@@ -0,0 +1,70 @@
+use pgrx::*;
+use serde_json::json;
+use std::env;
+use std::fs;
+
+pub unsafe fn init() {
+    // Read whether telemetry was already handled at the ParadeDB level
+    let telemetry_handled = match fs::read_to_string("/tmp/telemetry") {
+        Ok(content) => content.trim().to_string(),
+        Err(_) => String::from("false"),
+    };
+
+    // If telemetry was not already handled at the ParadeDB level, this is a standalone deployment
+    // of pg_search, and we should send telemetry
+    if telemetry_handled != "true" {
+        // We only send telemetry in the standalone deployment if TELEMETRY is set to true
+        let telemetry = env::var("TELEMETRY").unwrap_or_else(|_| String::from("false"));
+        if telemetry == "true" {
+            if let Ok(api_key) = env::var("POSTHOG_API_KEY") {
```

Comment:
Could we clean this up? Flatten the `ifs` by breaking if the keys don't exist

Reply:
You can clean it up further with the `any` operator

Reply:
done ^